### ISSUE TITLE
drop "Contributed-under" from the unkempt git guide (Bugzilla Bug 1681)

### ISSUE THREAD

**Issue by tianocore-issues** [tianocore/tianocore.github.io#41]
This issue was created automatically with [bugzilla2github](https://github.com/makubacki/bugzilla2github/tree/bz2gh_tianocore)
# Bugzilla Bug 1681
Date: 2019-04-03T08:20:35+00:00
From: @lersek
To:   @lersek
CC:   @lersek, @mdkinney

Duplicates:   #1656
Depends on:   #1373
Last updated: 2019-04-04T12:08:08+00:00

**Comment by tianocore-issues** [comment]
## Comment 6988
Date: 2019-04-03 08:20:35 +0000
From: @lersek

- Industry Specification: ---
- Target OS: ---
- Bugzilla Assignee(s): @lersek

Step &lt;https://github.com/tianocore/tianocore.github.io/wiki/Laszlo&apos;s-unkempt-git-guide-for-edk2-contributors-and-maintainers#contrib-07&gt; says that contributors should create a commit message template that ends with:

&gt; Contributed-under: TianoCore Contribution Agreement 1.1
&gt; Signed-off-by: Your Name &lt;Your Email Address&gt;

This will no longer be correct once bug 1373 is fixed, so remove the recommendation at that point.

**Comment by tianocore-issues** [comment]
## Comment 7004
Date: 2019-04-04 11:43:20 +0000
From: @mdkinney

I think this is a duplicate of:

https://bugzilla.tianocore.org/show_bug.cgi?id=1656

**Comment by tianocore-issues** [comment]
## Comment 7005
Date: 2019-04-04 12:08:08 +0000
From: @lersek

Correct: I missed the last sentence in &lt;https://bugzilla.tianocore.org/show_bug.cgi?id=1656#c0&gt;. Thanks!

*** This bug has been marked as a duplicate of bug 1656 ***
